apps/subscribe-button/src/overlay/clients/Platforms.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <h1>{{ $t('PLATFORM.TITLE') }}</h1>
+    <ul>
+      <li v-for="(c, index) in platform_apps" :key="index">
+        <!-- <b><img :src="`/static/img/${osx.icon}`" />{{ osx.title }}</b> -->
+        <!-- <a :href="`${osx.scheme}https://freakshow.fm/feed/m4a`" target="_blank">{{ osx.title }}</a> -->
+        <a :href="`${c.scheme}`" target="_blank">{{ c.title }}</a>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import platform from './clientlist/platform.json'
```

Comment:
Am besten die Platformen in den store packen und selektieren, dann kannst du später die infos aus unterschiedlichen Quellen ziehen.